packages/fiori/src/SideNavigation.ts

```diff
@@ -51,20 +52,22 @@ type SideNavigationPopoverContents = {
 };
 
 type SideNavigationSelectionChangeEventDetail = {
-	item: SideNavigationItemBase;
+	item: SideNavigationItemBase,
 };
 
 // used for the inner side navigation used in the SideNavigationPopoverTemplate
 type PopupClickEventDetail = {
 	target: {
```

Comment:
Instead of creating type for the whole event, create type that describes only the target which is something like:

SideNavigationItem & { associatedItem: SideNavigationItemBase}

and in `handlePopupItemClick` method cast the e.target as type newly created type